src/rpc-browser-ws.ts

```diff
@@ -27,6 +30,7 @@ export class RpcBrowserWebSockets extends RpcCommon {
     setTimeout(() => {
       const promiseCallbacks: IPromiseCallbacks | undefined = this.promiseCallbacks.get(id);
       if (promiseCallbacks) {
+        this.logger.warn(`Request ${id} method ${method} has timed out`);
```

Comment:
It is preferable to use json with parameters as the second parameter.
this.logger.warn(`Request ${id} method ${method} has timed out`, {id, method});

Reply:
as discussed, we will extract parts of the message into logging fields on demand, no need for it for now